### DISCUSSION TITLE
No node visible in node-RED left pane

### DISCUSSION THREAD

**Discussion by DIY3D** [wz2b/node-red-contrib-influxdb-line-protocol#10] [Q&A]
Hi 
node-RED is running on a Pi, 
I have installed the Influx-line-protocol in node-RED (Manage Pallete > Install Tab).
Under Manage Pallete > Nodes Tab I see @wz2b/node-red-influxdb-line-protocol installed, strange thing is I can't find the node in my left pane in node-RED. Do you have any idea what's going on here?

regards,
Marco

**Comment by agentsmith1612** [comment]
I searched for it, too.

I found that the node on the left is labeled as "InfluxDB / Json".
